ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -599,9 +599,59 @@
           started_status: null
           wrappers: []
           publishers: []
+      - '{name}-{test_name}-no-scm':
+          test_name: windows-e2e-proxyall-pending-label
```

Comment:
We should remove all windows proxyall jobs, `proxyall` is enabled by default now for windows agent.